sdk/appconfiguration/azure-spring-cloud-feature-management/README.md

```diff
@@ -259,6 +259,171 @@ Options are available to customize how targeting evaluation is performed across
     }
 ```
 
+## Dynamic Features
+
+Using `FeatureManager`'s `getVariantAsync` method allows for returning of dynamic features based off a feature evaluator. Usage:
+
+```java
+featureManager.getVariantAsync("DiscountBanner", DiscountBanner.class)
+```
+
+This example shows requesting a Dynamic Feature named "DiscountBanner", it requires the name of the Dynamic Feature and the type that needs to be returned.
+
+### Defining a Dynamic Feature
+
+A Dynamic Feature can be defined in the local configuration file using the following format.
+
+```yaml
+feature-management:
+  dynamic-features:
+    DiscountBanner:
+      assigner: "Microsoft.Targeting"
+      variants:
+        -
+          name: Big
+          default: true
+          configuration-reference: "DiscountBanner:Big"
+          assignment-parameters:
+            ...
+        -
+          name: Small
+          default: false
+          configuration-reference: "DiscountBanner:Small"
+          assignment-parameters:
+            ...
+feature-variant:
+  DiscountBanner:
+    Big:
+      size: 400
+      color: "#BBB"
+    Small:
+      size: 150
+      color: "#DDD"
+```
+
+A Dynamic Feature has two main parts, the assigner and a list of variants. The assigner in this case `feature-management.DiscountBanner.assigner` which is pointing the `TargetingEvaluator` `Microsoft.Targeting` that will evaluate the list of variants and choose which configuration-reference will be used. Each variant has: a name, configuration-reference, and assignment-parameters. One of the listed variants needs to be configured as the default variant for the feature. The name field is meta data to reference the variant and just needs to be unique. The configuration-reference is a reference to the configurations defined under `feature-variant` in the configuration file. **Note:** the colon notation is required when denoting parts of the configuration. If it is needed multiple colons can be used to reference sub configurations. i.e. `DiscountBanner:Big:size` would return the `Integer` 400.
```

Comment:
The assigner in this case `feature-management.DiscountBanner.assigner` which is pointing the `TargetingEvaluator` `Microsoft.Targeting` that will evaluate the list of variants and choose which configuration-reference will be used.

Looks like it's missing some words?

The assigner used by a dynamic feature is specified by the `feature-management.DiscountBanner.assigner` property. In this example `Microsoft.Targeting` is specified which means that the `TargetingEvaluator`  will be used.

Reply:
It's best to avoid fillers like "just" in technical writing.

Reply:
metadata is one word.

Reply:
I see assignment parameters is separated into a separate paragraph. Why not just do a bulleted list to describe all of their properties in their own bullet?

Reply:
The description seems a bit long for a bulleted list.

Reply:
The description of each property is too long?

Reply:
I just think bulleted lists are only good for short items.

Reply:
I don't disagree. I'm just wondering which property description isn't a short item?